registry/registry.go

```diff
@@ -107,6 +116,11 @@ func (c *Client) GetRepository(repository string) ([]flux.ImageDescription, erro
 	}
 
 	tags, err := client.Tags(hostlessImageName)
+	c.Metrics.RequestCount.With(
+		LabelRepository, repository,
+		LabelRequestKind, RequestKindTags,
+		LabelSuccess, strconv.FormatBool(err == nil),
+	).Add(1)
```

Comment:
Maybe request duration histograms? then we can get count as well, I believe? Would let us break down the FetchDuration above into "tags" and "metadata" requests.